src/pages/apply.js

```diff
@@ -54,6 +54,7 @@ class ApplicationForm extends Component {
     sending: false,
     submitted: false,
     alreadyApplied: false,
+    blackListed: false,
```

Comment:
We should use the terms allowlist/denylist going forward as whitelist/blacklist is racially insensitive (if not in this PR we should track a codebase cleanup in an issue). Worst case it's a minor inconvenience to some, best case it makes the code more inclusive.

Reply:
https://github.com/BUUPE/interview/issues/128 - Issue opened (I'll probably tackle this later today just to fix it before it spreads)